### DISCUSSION TITLE
SimpleLogin sold

### DISCUSSION THREAD

**Discussion by Metabaron1** [simple-login/app#890] [Q&A] [answered]
Hi
https://techcrunch.com/2022/04/08/proton-buys-simplelogin/
You must be rich now, good job  ;) 

Will you continue to maintain SL as open source and with self hosted releases?
I mean, any side effects for self hosted users at long term?

**Comment by 0xspade** [comment]
I hope that they update the self-hosted.

**Reply by nguyenkims** [reply]
Yes it's definitely in our todo list.

**Comment by PureFallen** [comment]
SimpleLogin and ProtonMail confirmed the joining in their corresponding blogposts:
https://simplelogin.io/blog/simplelogin-join-proton/
https://protonmail.com/blog/proton-and-simplelogin-join-forces/

The blog posts reveal that SimpleLogin is planned to continue open source. But everything seems to be in very early stages. No comment on when the open source approach is for auditing purposes or if Selfhosting will continue to be an option.
Depending on how deep the integration into ProtonMail will look like i kinda wonder if there is going to be benefits/discounts for those you are paying for both services.

**Comment by nguyenkims** [comment, marked as answer]
> Will you continue to maintain SL as open source and with self hosted releases?

Yes.

> No comment on when the open source approach is for auditing purposes or if Selfhosting will continue to be an option.
Depending on how deep the integration into ProtonMail will look like i kinda wonder if there is going to be benefits/discounts for those you are paying for both services.

We'll publish the audit result soon. SimpleLogin has been run through a security audit and we're fixing all the (minor) issues raised in the report before publishing the report.

It isn't defined yet but there's going to be a price bundle for some Proton plans to include SimpleLogin.
